<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fm_grupos</class>
 <widget class="QDialog" name="fm_grupos">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Grupos de segurança - Agenda de contatos</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource_files.qrc">
    <normaloff>:/ico/images/icone.png</normaloff>:/ico/images/icone.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Pesquisar:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_pesquisar"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_pesquisar">
       <property name="text">
        <string>Pesquisar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_mostrar_todos">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Mostrar todos</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QTableWidget" name="tableWidget_grupos"/>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>ID:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_id">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Descrição:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_descricao"/>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Sigla:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_sigla"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_salvar">
       <property name="text">
        <string>Salvar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_limpar">
       <property name="text">
        <string>Limpar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_selected_row">
       <property name="text">
        <string>-1</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_fechar">
       <property name="text">
        <string>Fechar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEdit_pesquisar</tabstop>
  <tabstop>pushButton_pesquisar</tabstop>
  <tabstop>pushButton_mostrar_todos</tabstop>
  <tabstop>tableWidget_grupos</tabstop>
  <tabstop>lineEdit_id</tabstop>
  <tabstop>lineEdit_descricao</tabstop>
  <tabstop>comboBox_sigla</tabstop>
  <tabstop>pushButton_salvar</tabstop>
  <tabstop>pushButton_limpar</tabstop>
  <tabstop>pushButton_fechar</tabstop>
 </tabstops>
 <resources>
  <include location="resource_files.qrc"/>
 </resources>
 <connections/>
</ui>
